web/server/vue-cli/src/mixins/confidentiality.mixin.js

```diff
@@ -0,0 +1,48 @@
+import { Confidentiality } from "@cc/prod-types";
+
+export default {
+  data() {
+    return {
+      confidentialityIconDefs: {
+        "Confidential classified": "mdi-shield-lock-outline",
+        "Internal classified": "mdi-shield-home-outline",
+        "Open classified": "mdi-shield-sun-outline",
+      }
```

Comment:
Use Confidentiality enum values instead of strings:
```suggestion
      confidentialityIcons: {
        Confidentiality.CONFIDENTIAL: "mdi-shield-lock-outline",
        Confidentiality.INTERNAL: "mdi-shield-home-outline",
        Confidentiality.OPEN: "mdi-shield-sun-outline",
      }
```

Reply:
I think this snippet has a syntax error, but I eliminated the duplicated keys, with a different method.